cmd/otelcol/Dockerfile.windows2022

```diff
@@ -0,0 +1,44 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
```

Comment:
Since only the tag is different would it make sense to use a build arg until version specific differences lead us to another file?